hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieMergeHandle.java

```diff
@@ -102,6 +118,15 @@
   protected Map<String, HoodieRecord<T>> keyToNewRecords;
   protected Set<String> writtenRecordKeys;
   protected HoodieFileWriter<IndexedRecord> fileWriter;
+  // a flag that indicate whether allow the change data to write out a cdc log file.
+  protected boolean cdcEnabled = false;
+  protected String cdcSupplementalLoggingMode;
```

Comment:
Why is this a String?

Reply:
do you suggest that should use an enum?

Reply:
Was reviewing this before i caught up w/ an updated version of the RFC so got confused.
Yeah, let's use enum for this one